.gitignore

```diff
@@ -1,3 +1,4 @@
 /openwrt
 /keys
 /cloud/app/log/
+tags
```

Comment:
Ctags generates this. This is what's commonly used in vim for C development. If it's an issue we can remove it.

Reply:
What would like us to do with that one?

Reply:
no one in the history of openwrt has ever required this. It would be a patch that we carry around with us forever. Ideally you submit it upstream, and then it will treacle down to tip with the next owrt rebase

Reply:
I'll just remove it then